source/dub/generators/generator.d

```diff
@@ -590,7 +590,7 @@ struct GeneratorSettings {
 	bool filterVersions;
 
 	// only used for generator "build"
-	bool run, force, direct, rdmd, tempBuild, parallelBuild;
```

Comment:
can you add a line of documentation to your new field? I think it's the least obvious out of all fields here.

Reply:
Fixed :)